### PR TITLE
Admin UI redesign

### PR DESCRIPTION
**Pull request by jsorohova**

### CODE REVIEW

File: src/themes/designer/styles/scaffolding.scss [hunk 1]
```diff
@@ -9,7 +9,7 @@ body {
 
 * {
     @include box-sizing(border-box);
-    position: relative;
+    //position: relative;
```
Comment (ygrik) [inline src/themes/designer/styles/scaffolding.scss:12]: does this affect the existing editor styles?

File: src/themes/designer/styles/styles.scss [hunk 2]
```diff
@@ -40,4 +41,7 @@
 @import "editors/colorPicker.scss";
 
 
-@import "grid.scss";
\ No newline at end of file
+@import "grid.scss";
+
+// TODO: adjust webpack to import from the library
```
Comment (ygrik) [inline src/themes/designer/styles/styles.scss:46]: what adjustment need to be done?

File: src/themes/website/images/nature.jpg
Comment (ygrik) [inline src/themes/website/images/nature.jpg:1]: do we require this file in sources?

**Review by ygrik — CHANGES_REQUESTED** [review]

File: src/admin/rightPanel.tsx [hunk 3]
```diff
@@ -0,0 +1,335 @@
...
+                    iconName={option.data.icon}
+                    title={option.data.icon}
+                />
+            )}
+            <span>{option.text}</span>
+        </Stack>
+    )
+
+    // This is added because using renderDropdownOption is not working in some cases
+    // TODO: fix this duplicate function
```
Comment (JMach1) [inline src/admin/rightPanel.tsx:189]: forgotten TODO?

File: src/admin/utils/components/backButton.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+const backIcon: IIconProps = { iconName: 'ChevronLeftMed' };
+
+interface BackButtonProps {
+    onClick: () => void
+}
+
+export class BackButton extends React.Component<BackButtonProps, {}> {
+    constructor(props: BackButtonProps) {
+        super(props);
+    }
```
Comment (JMach1) [inline src/admin/utils/components/backButton.tsx:13]: I think this empty constructor is not needed?

File: src/admin/utils/components/deleteConfirmationOverlay.tsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+interface DeleteConfirmationProps {
+    deleteItemTitle: string,
+    onConfirm: () => void,
+    onDismiss: () => void
+}
+
+export class DeleteConfirmationOverlay extends React.Component<DeleteConfirmationProps, {}> {
+    constructor(props: DeleteConfirmationProps) {
+        super(props);
+    }
```
Comment (JMach1) [inline src/admin/utils/components/deleteConfirmationOverlay.tsx:33]: empty constructor which can be removed

File: src/admin/utils/components/labelWithInfo.tsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+interface LabelWithInfoProps {
+    label: string,
+    info: string
+}
+
+export class LabelWithInfo extends React.Component<LabelWithInfoProps, {}> {
+    constructor(props: LabelWithInfoProps) {
+        super(props);
+    }
```
Comment (JMach1) [inline src/admin/utils/components/labelWithInfo.tsx:12]: empty constructor which can be removed

File: src/admin/utils/components/toastNotification.tsx [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+interface ToastNotificationProps {
+    title: string,
+    description: string
+}
+
+export class ToastNotification extends React.Component<ToastNotificationProps, {}> {
+    constructor(props: ToastNotificationProps) {
+        super(props);
+    }
```
Comment (JMach1) [inline src/admin/utils/components/toastNotification.tsx:13]: empty constructor which can be removed

**Pull request merged**